### PR TITLE
UPDATE angular to v5 && UPDATE angular-cli to latest

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: -cli tests fail-

## What I did
Well I attempted to fix by updating angular and the angular-cli to latest versions, it seems to change the behavior, so it's worth a shot. Maybe this fixes things.

Also added a flag to the typescript compiler options to: https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#new---skiplibcheck

## How to test
- run the bootstrapping sequence for core
- run the angular example (this seems to throw some warnings but it does function correctly)
- run the test including cli

### CODE REVIEW

File: lib/cli/test/fixtures/angular-cli/package.json [hunk 1]
```diff
@@ -12,24 +12,24 @@
...
-    "@angular/common": "^2.3.1",
-    "@angular/compiler": "^2.3.1",
-    "@angular/core": "^2.3.1",
-    "@angular/forms": "^2.3.1",
-    "@angular/http": "^2.3.1",
-    "@angular/platform-browser": "^2.3.1",
-    "@angular/platform-browser-dynamic": "^2.3.1",
+    "@angular/common": "^5.0.0",
+    "@angular/compiler": "^5.0.0",
+    "@angular/core": "^5.0.0",
```
Comment (Hypnosphi) [inline lib/cli/test/fixtures/angular-cli/package.json]: Does this change mean that we don't support apps which use earlier versions of angular? We should reflect it in peerDeps than and do a version check in `detect.js`

**Review by Hypnosphi — CHANGES_REQUESTED** [review]

File: lib/cli/test/fixtures/angular-cli/package.json (hunk 1 above)
Comment (Hypnosphi) [reply lib/cli/test/fixtures/angular-cli/package.json]: Before doing that, we should probably find out the minimal supported version (and use it in fixture)

Comment (ndelangen) [reply lib/cli/test/fixtures/angular-cli/package.json]: It MIGHT work 🕶

Comment (Hypnosphi) [reply lib/cli/test/fixtures/angular-cli/package.json]: The tests on `release/3.3` are fixed, but there's still a value in upgrading dependensies in app and example. Please just remove all the changes in CLI fixtures, and revert https://github.com/storybooks/storybook/pull/2208/commits/7bad0c0a26619b6dea99790a4070c3192698756b

Comment (alterx) [reply lib/cli/test/fixtures/angular-cli/package.json]: Yup, Angular itself should be backwards compatible. The minimum supported should be 2.3.1 (the one that's inside the fixtures)

Comment (danielduan) [reply lib/cli/test/fixtures/angular-cli/package.json]: @alterx can you please take a look at the current 3.3 branch and see if you can fix the angular cli problems in there?

I don't know much about angular but the CLI test itself is breaking and throwing errors. It's preventing us from testing other things and releasing the next alpha version.

Comment (alterx) [reply lib/cli/test/fixtures/angular-cli/package.json]: Getting 
```
 ~/Projects/storybook   release/3.3  npm run test

> storybook@3.0.0 test /Users/carlosvegarodriguez/Projects/storybook
> ./scripts/test.js

? Select which tests to run Core & React & Vue (core), React-Native example (react-native-vanilla)
● Validation Error:

  Module <rootDir>/node_modules/react-native/jest/setup.js in the setupFiles option was not found.

  Configuration Documentation:
  https://facebook.github.io/jest/docs/configuration.html
```
When trying to run the tests

Comment (Hypnosphi) [reply lib/cli/test/fixtures/angular-cli/package.json]: Either don't select react-native option in test, or run bootstrap for react-native before running it

Comment (Hypnosphi) [reply lib/cli/test/fixtures/angular-cli/package.json]: @danielduan Are you talking about the issue I fixed yesterday?

**Pull request closed without merging**